Sources/Mockolo/Executor.swift

```diff
@@ -132,6 +133,9 @@ class Executor {
         useSourceKit = parser.add(option: "--use-sourcekit",
                              kind: Bool.self,
                              usage: "Whether to use SourceKit for parsing. By default it will use SwiftSyntax.")
+        enableArgsHistory = parser.add(option: "--enable-args-history",
+                                       kind: Bool.self,
+                                       usage: "Whether generated mock contains function args history capturing. If set, `history` annotation will be ignored (default = false).")
```

Comment:
"Whether to enable args history for all functions (default = false).  To enable history per function, use the 'history' keyword in the annotation argument. "

Reply:
Fixed this and README in 2164945.